sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_internal/client_base.py

```diff
@@ -55,7 +54,7 @@ def __init__(self, vault_url, credential, **kwargs):
                 return
 
             pipeline = kwargs.pop("pipeline", None)
-            transport = kwargs.pop("transport", RequestsTransport(**kwargs))
+            transport = kwargs.pop("transport", None)
```

Comment:
After double-checking to make sure it won't break anything, I think we can actually remove the `transport` and `pipeline` variables here, instead of fetching them and passing them to the `_KeyVaultClient` below. If these keyword arguments are passed, they'll be used appropriately when the `PipelineClient` that uses them is created down the line.

Reply:
@mccoyp I'm assuming this applies for the other SDKs too?

Reply:
Yep! Same for the other comment